modules/accredited_representative_portal/app/services/accredited_representative_portal/power_of_attorney_request_service/accept.rb

```diff
@@ -92,7 +92,7 @@ def form_payload
         {}.tap do |a|
           a[:veteran] = veteran_data
           a[:serviceOrganization] = organization_data
-          a[:recordConsent] = form_data.dig('authorizations', 'recordDisclosure')
+          a[:recordConsent] = form_data.dig('authorizations', 'recordDisclosureLimitations').blank?
```

Comment:
What are the options for `recordDisclosureLimitations`? Since it's a behavior change, I just want to check

Reply:
@ojbucao Just a reminder this PR is still around open.

Reply:
@LindseySaari it's basically the things they want to keep private. The list can be seen [here in this spec](https://github.com/department-of-veterans-affairs/vets-api/blob/cf938d08dd7504034092825636ea9996588fc671/modules/representation_management/spec/services/power_of_attorney_request_service/orchestrate_spec.rb#L100)